requirements.txt

```diff
@@ -1,6 +1,7 @@
 icalendar>=3.10
 python-dateutil>=2.5.2
 python-hglib>=2.6.1
+pytz==2024.2
```

Comment:
As for the other requirements, let's use `>=` rather than `==` (and I guess an older version would work too, let's try `>=2022`)